### ISSUE TITLE
Auto immunity File number 3

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#7044]
See the attached file.
Many thanks,
Maud

[20210715 FAI2R_HM final.xlsx](https://github.com/obophenotype/human-phenotype-ontology/files/6849553/20210715.FAI2R_HM.final.xlsx)

**Comment by pnrobinson** [comment]
@matentzn -- can we process this file in a similar way as we did a few months ago? That would be amazing :smile:

**Comment by matentzn** [comment]
Yes, but I would suggest that from now on, we ask the terms to be submitted in that format - its super simple, and even if a bit broken, we can fix it quickly. I this case here, do you think we (as HPO team) should be preparing the table or can we ask @MaudDLVT to do it? I can help with the process of course, and I can also do it - but in the interest of decentralisation it may be worth just training the community to do it themselves.

**Comment by pnrobinson** [comment]
@matentzn  It is very rare to have so many analogous terms like this -- I cannot think of any other example. The vast majority of grouped requests are much more heterogeneous where a form like this would not make much sense. But once we hopefully have the new grant we should definitely discuss improving the way we work with the community -- topic for a huddle?

**Comment by matentzn** [comment]
Ok.

Here is the table:
https://docs.google.com/spreadsheets/d/1dCU7UCkUp1TLFtsxqq2yXK_AhR5FsPMZtZq3d6TOyTU/edit#gid=0

To do:
- [ ] Delete the `LABEL` wherever you dont want the label to be overwritten (probably not necessary - only of @MaudDLVT  actually changed the label of an existing HPO id).
- [ ] Where should the PMIDs go in the absensce of a textual definition?
- [ ] Where should the ORCIDs go? Usually these are on the text definitions or synonyms, but there are no text definitions in this spreadsheet. There are some synonyms, but not enough to get the ORCIDs on all terms.

Let me know if you are happy with the spreadsheet, I will then create the pull request on HPO.

**Comment by MaudDLVT** [comment]
Hi @matentzn, is that demand still pending? 
Many thanks, Maud 
(This table is the same as last time, do I have to change things?)

**Comment by matentzn** [comment]
No this is up to @pnrobinson to deal with. Sorry about the delay!

@pnrobinson I am ready to make a pull request when you are!

**Comment by pnrobinson** [comment]
@matentzn I do not think these are new terms, these seem to be additions to existing terms.
@MaudDLVT can you confirm that we need to update these existing terms? I do not understand the action item here.

**Comment by matentzn** [comment]
We need answers to these questions:

1. Where should the PMIDs go in the absensce of a textual definition?

2. Where should the ORCIDs go? Usually these are on the text definitions or synonyms, but there are no text definitions in this spreadsheet. There are some synonyms, but not enough to get the ORCIDs on all terms.

3. What should I do if the LABEL in spreadsheet differs from the label in HPO?

**Comment by MaudDLVT** [comment]
@pnrobinson and @matentzn no need to update the existing terms (maybe the PMID?). Do I have to delete the corresponding rows?

**Comment by matentzn** [comment]
Yes, if you do not want the `LABEL` to be changed in HPO, delete it. So they are all existing terms?

**Comment by MaudDLVT** [comment]
@matentzn I removed the 17 existing terms.

**Comment by matentzn** [comment]
Thanks. Still need @pnrobinson to answer the other two questions before finalising.

**Comment by MaudDLVT** [comment]
@pnrobinson, for:
@MaudDLVT can you confirm that we need to update these existing terms? I do not understand the action item here.
=> It is only new terms now, I removed all existing terms. Many thanks!

**Comment by pnrobinson** [comment]
I am adding this by hand.
Still to go

- [ ] Anti-phosphatidyl choline antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstphosphatidyl choline	PMID:31354742	aPTC	ORCID:0000-0002-3387-1836
- [ ] Anti-phosphatidyl glycerol antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstphosphatidyl glycerol	PMID:26563139	aPG	ORCID:0000-0002-3387-1836
- [ ] Anti-phosphatidyl inositol antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstphosphatidyl inositol	PMID:31354742	aPI	ORCID:0000-0002-3387-1836
- [ ] Anti-phosphatidyl serine antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstphosphatidyl serine	PMID:26563139	aPS	ORCID:0000-0002-3387-1836
- [ ] Anti-annexin-V antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstannexin-V	PMID:33464429		ORCID:0000-0002-3387-1836
- [ ] Anti-sphingolipid antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstsphingolipid	PMID:31354742	aSL	ORCID:0000-0002-3387-1836
- [ ] Anti-myeloperoxidase antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstmyeloperoxidase	PMID:14646155	Anti-MPO antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-PR3 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstPR3	PMID:14646155		ORCID:0000-0002-3387-1836
- [ ] Anti-Neutrophil elastase antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstNeutrophil elastase	PMID:32091008		ORCID:0000-0002-3387-1836
- [ ] Anti-neutrophil cytoplasmic antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstneutrophil cytoplasmic	PMID:14646155	ANCA	ORCID:0000-0002-3387-1836
- [ ] Anti-transcription intermediary factor-1gamma antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againsttranscription intermediary factor-1gamma	PMID:33876712	Anti-TIF-1gamma antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-Su antigen/Argonaute 2 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstSu antigen/Argonaute 2	PMID:33876712	Anti-Su antibody positivity, Anti-Ago2 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-p53 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstp53	PMID:29554146		ORCID:0000-0002-3387-1836
- [ ] Anti-Y-box protein-1 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstY-box protein-1	PMID:29554146	Anti-YB 1 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-GW182 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGW182	PMID:14598044|PMID:29554146		ORCID:0000-0002-3387-1836
- [ ] Anti-Ge-1 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstGe-1	PMID:29554146	Anti-Gerbich phenotype 1 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-MIT3 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstMIT3	PMID:29554146|PMID:32286240		ORCID:0000-0002-3387-1836
- [ ] Anti-gp210 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstgp210	PMID:29554146|PMID:32286240		ORCID:0000-0002-3387-1836
- [ ] Anti-sp100 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstsp100	PMID:29554146|PMID:32286240		ORCID:0000-0002-3387-1836
- [ ] Anti-type I liver-kidney microsomal antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againsttype I liver-kidney microsomal	PMID:29554146	Anti-LKM1 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-Hexokinase-1 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstHexokinase-1	PMID:29554146	Anti-HK-1 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-Kelch like protein 12 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstKelch like protein 12	PMID:29554146	Anti-KLHL-12 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-tyrosine phosphatase-related islet antigen 2 antibody	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againsttyrosine phosphatase-related islet antigen 2 antibody	PMID:32459722	Anti-IA2 antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-insulin antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstinsulin	PMID:32459722	Anti-IAA antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-thyroglobulin antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstthyroglobulin	PMID:32459722	Anti-TG antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-thyroperoxidase antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstthyroperoxidase	PMID:32459722	Anti-TPO antibody positivity	ORCID:0000-0002-3387-1836
- [ ] Anti-thyrotropin receptor antibody	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstthyrotropin receptor antibody	PMID:32459722	TRAB	ORCID:0000-0002-3387-1836
- [ ] Anti-angiotensin-converting enzyme 2 antibody positivity	The presence of autoantibodies (immunoglobulins) in the blood circulation that react againstangiotensin-converting enzyme 2	PMID:33910447	Anti-ACE2 antibody positivity	ORCID:0000-0002-3387-1836

**Comment by matentzn** [comment]
I could have done it real quick!

**Comment by pnrobinson** [comment]
@matentzn I know, I was worried that we have gotten some things duplicate since I took too long to process, and also wanted to revise some of the items...thanks!

**Comment by pnrobinson** [comment]
@MaudDLVT  Is [Anti-multiple nuclear dots antibody positivity](https://hpo.jax.org/app/browse/term/HP:0025353) the same thing as Anti-sp100 antibody positivity? Not adding the latter for now

**Comment by pnrobinson** [comment]
@MaudDLVT  all done except for Anti-sp100 antibody positivity -- should this be added as a synonym to Anti-multiple nuclear dots antibody positivity  ?

**Comment by MaudDLVT** [comment]
> 

Question asked to to physicians. Will let you know as soon as possible.

**Comment by MaudDLVT** [comment]
> @MaudDLVT all done except for Anti-sp100 antibody positivity -- should this be added as a synonym to Anti-multiple nuclear dots antibody positivity ?

@pnrobinson Anti-sp100 might be a subtype of Anti-multiple nuclear dots antibody positivity HP:0025353. "Autoantibodies causing the MND pattern are directed against sp100, sp140, promyelocytic leukemia nuclear body proteins (PML) or ubiquitin-like modifier (SUMO) proteins, while in the punctate nuclear envelope pattern, the target antigens are gp210, nucleoporin-62 or laminin B receptor proteins."

**Comment by pnrobinson** [comment]
Added Anti-sp100 as suggested @MaudDLVT
